docs/user-guide/src/bmo/automated_cleaning.md

```diff
@@ -0,0 +1,32 @@
+# Automated Cleaning
+
+One of the Ironic's feature exposed to Metal3 Baremetal Operator is [node automated cleaning](https://docs.openstack.org/ironic/latest/admin/cleaning.html#automated-cleaning). When enabled, automated cleaning kicks off while a node is in the first provisioning and on every deprovisioning.
+
+There are two automated cleaning modes available which can be set via `automatedCleaningMode` field of a BareMetalHost `spec`.
+
+- `metadata` to enable the cleaning
```

Comment:
I think this deserves an explanation. Why is it `metadata` to enable instead of just `enable`?

Reply:
Added.